src/main/java/com/example/demo/domain/MemberImpl.java

```diff
@@ -54,4 +57,9 @@ public void addStudyMember(StudyMember studyMember) {
 		studyMembers.add(studyMember);
 	}
 
+	public void updateProfile(ProfileUpdateRequest profileUpdateRequest) {
```

Comment:
[제안] 엔티티가 dto를 의존하고 있네요
String 타입으로 받고, 안에 검증로직을 넣으면 될것같아요
```suggestion
	public void updateProfile(String introduction) {
	//update 검증로직 추가
	}
```

Reply:
> [제안] 엔티티가 dto를 의존하고 있네요 String 타입으로 받고, 안에 검증로직을 넣으면 될것같아요

이 부분 기존에 어떻게 처리했었는지 가물가물해서, 편한대로 구현했어요. 다른 기능 구현에서 어떻게 했는지 확인부터 해봐야 하겠네요. 다른 부분들도 String으로 분리했다면 일관성을 지키기 위해 그렇게 분이하는 것이 나을 것 같아요.